tests/integrations/realcluster/real_cluster.go

```diff
@@ -35,7 +35,10 @@ type realClusterSuite struct {
 	suiteName  string
 }
 
-var tiupBin = os.Getenv("HOME") + "/.tiup/bin/tiup"
+var (
+	playgroundLogDir = filepath.Join("tmp", "real_cluster", "playground")
+	tiupBin          = os.Getenv("HOME") + "/.tiup/bin/tiup"
```

Comment:
Is it possible to use command like `which tiup` to detect the real path here rather than hard-coded?